data/reusables/reminders/slack-channel.md

```diff
@@ -1 +1,3 @@
 1. Under "Slack channel", type the name of the Slack channel where you'd like to receive notifications.
+   > [!TIP]
+   > If this Slack channel is private, you will need to invite the integration into the channel: `/invite @github`. In addition, you need to ask users to run `/github signin` in the Slack channel otherwise they will not be @mentioned. For more information, see [Getting started](https://github.com/integrations/slack?tab=readme-ov-file#getting-started) in the Slack integrations documentation.
```

Comment:
```suggestion
   > If this Slack channel is private, you will need to invite the integration into the channel: `/invite @github`. In addition, you need to ask users to run `/github signin` in one of their Slack channels, otherwise they will not be @mentioned. For more information, see [Getting started](https://github.com/integrations/slack?tab=readme-ov-file#getting-started) in the Slack integrations documentation.
```